lib/babe/build_integration_test.go

```diff
@@ -374,18 +332,23 @@ func TestBuildBlockTimeMonitor(t *testing.T) {
 		Authority: true,
 	}
 
-	babeService := createTestService(t, cfg)
+	genesis, genesisTrie, genesisHeader := newWestendDevGenesisWithTrieAndHeader(t)
+	babeService := createTestService(t, cfg, genesis, genesisTrie, genesisHeader)
```

Comment:
is `cfg` even needed anymore?

Reply:
I'm not entirely sure, there are a couple of things to consider.
1) Config and the VerificationManager were both used and weirdly coupled in the verification manager integrations test. My plan was in my follow-up PR to investigate this further and find out what we need and what we don't for the tests to work and be intuitive.
2) Even if we don't use it now, perhaps we would want to in the future. I think this still relates to my first point though